interface/patient_file/transaction/add_transaction.php

```diff
@@ -146,18 +146,12 @@ function end_group() {
 ?>
 <html>
 <head>
-<?php html_header_show(); ?>
```

Comment:
I wouldn't remove the above, since it seems that some folks use this as a hook for custom stuff.